client/src/components/guests.js

```diff
@@ -5,34 +5,95 @@ import './Guests.css'
 
 class Guests extends React.Component {
```

Comment:
This was a great idea, i see you breaking out a lot of stuff that used to live in the App component and moving them to this component